templates/automatic-release/release.config.js

```diff
@@ -1,24 +1,14 @@
 module.exports = {
   "branches": [
     "main",
-    "main-built",
     "next",
```

Comment:
Did you test leaving next here?

Reply:
Yes I tested, and before the change we introduced with the `-built` branches, the file was like this. I believe it is working because the `next` branch works only to build the next version release, and maybe without an automatic trigger of the release workflow it can have a good reason to stay here.